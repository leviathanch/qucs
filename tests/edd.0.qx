load legacy
load legacy/components
load legacy/qucsator
load plugins/misc
load plugins/verilog

verilog
paramset EDD_ ModelFactory;
  .dev_type = "EDD_";
endparamset

options lang=leg_sch

<Components>
<EDD_ D1 1 0 100 -5 -11 0 0 "explicit" 0 "1" 0 "0" 1 "V1" 1>
<EDD_ D2 1 60 100 -5 -11 0 0 "explicit" 0 "1" 0 "0" 1 "V1" 1>
<EDD_ D3 1 120 100 -5 -11 0 0 "explicit" 0 "1" 0 "0" 1 "V1*2" 1>
<EDD_ D4 1 200 140 -5 -11 0 1 "explicit" 0 "2" 0 "0" 1 "V1" 1 "1" 1 "YY" 1>
<EDD_ D5 1 200 200 -5 -11 0 1 "explicit" 0 "2" 0 "0" 1 "V1" 1 "1" 1 "YY" 1>
</Components>

options lang=verilog_nl
list main
list EDD_*

options lang=qucsator
list main
list EDD_*
