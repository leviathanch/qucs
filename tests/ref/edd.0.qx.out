Qacs : Qucs alternative command shell
used for testing only
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
misc plugins: wip
legacy plugins: wip
module main();
EDD_1 #(.Type(explicit), .Branches(1), .I1(0), .Q1(V1)) D1(_net0, _net1);
EDD_1 #(.Type(explicit), .Branches(1), .I1(0), .Q1(V1)) D2(_net1, _net2);
EDD_2 #(.Type(explicit), .Branches(1), .I1(0), .Q1(V1*2)) D3(_net2, _net3);
EDD_3 #(.Type(explicit), .Branches(2), .I1(0), .Q1(V1), .I2(1), .Q2(YY)) D4(_net4, _net5, _net6, _net7);
EDD_3 #(.Type(explicit), .Branches(2), .I1(0), .Q1(V1), .I2(1), .Q2(YY)) D5(_net8, _net4, _net9, _net6);
endmodule // main

paramset EDD_1 ;
endparmset //EDD_1

paramset EDD_2 ;
endparmset //EDD_2

paramset EDD_3 ;
endparmset //EDD_3

# sckt main


# 15
EDD:D1 _net0 _net1 Type="explicit" Branches="1"  I1="D1.I1" Q1="D1.Q1"
  Eqn:EqnD1I1 D1.I1="0" Export="no"
  Eqn:EqnD1Q1 D1.Q1="V1" Export="no"
EDD:D2 _net1 _net2 Type="explicit" Branches="1"  I1="D2.I1" Q1="D2.Q1"
  Eqn:EqnD2I1 D2.I1="0" Export="no"
  Eqn:EqnD2Q1 D2.Q1="V1" Export="no"
EDD:D3 _net2 _net3 Type="explicit" Branches="1"  I1="D3.I1" Q1="D3.Q1"
  Eqn:EqnD3I1 D3.I1="0" Export="no"
  Eqn:EqnD3Q1 D3.Q1="V1*2" Export="no"
EDD:D4 _net4 _net5 _net6 _net7 Type="explicit" Branches="2"  I1="D4.I1" Q1="D4.Q1" I2="D4.I2" Q2="D4.Q2"
  Eqn:EqnD4I1 D4.I1="0" Export="no"
  Eqn:EqnD4Q1 D4.Q1="V1" Export="no"
  Eqn:EqnD4I2 D4.I2="1" Export="no"
  Eqn:EqnD4Q2 D4.Q2="YY" Export="no"
EDD:D5 _net8 _net4 _net9 _net6 Type="explicit" Branches="2"  I1="D5.I1" Q1="D5.Q1" I2="D5.I2" Q2="D5.Q2"
  Eqn:EqnD5I1 D5.I1="0" Export="no"
  Eqn:EqnD5Q1 D5.Q1="V1" Export="no"
  Eqn:EqnD5I2 D5.I2="1" Export="no"
  Eqn:EqnD5Q2 D5.Q2="YY" Export="no"
# paramset EDD_1
# paramset EDD_2
# paramset EDD_3
